Associations/rental.rb

```diff
@@ -0,0 +1,10 @@
+class Rental
+  def initialize(date, person, book)
+    @date = date
+    @book = book
```

Comment:
- [ ] For us to achieve the _belongs-to_ association for `rental` and `person`, we need to ensure `rental` and `person` each is adding itself to the `@book`, similar to the below code:

```ruby
def initialize(date, book, person)
    .
    .

    @book = book
    book.rentals << self

    @person = person
    person.rentals << self
  end
```

Let's ensure to improve upon this aspect.